src/components/ChannelSearch/SearchResults.tsx

```diff
@@ -94,23 +94,25 @@ const DefaultSearchResultItem = <
     const channel = result;
 
     return (
-      <div
+      <button
```

Comment:
Are there any styling issues with these button changes?

Reply:
There are indeed - I'll make these updates to stream-chat-css